components/01-atoms/text/text/07-code.twig

```diff
@@ -0,0 +1,25 @@
+ {#
+/**
+ * Available variables:
+ * - code__content - the content of the code (typically text)
+ *
+ * Available blocks:
+ * - - code_content - used to replace the content of the code with something other than text
+ *   for example: A formatted text field in Drupal
+ */
+#}
+
+{% set code__base_class = code__base_class|default('code') %}
+{% set code__attributes = code__attributes|default({}) %}
+{% set code__attributes = code__attributes|merge({
+  'class': bem(code__base_class, code__modifiers, code__blockname),
+}) %}
+
+<pre {{ bem(code__base_class, code__modifiers, code__blockname) }}>
```

Comment:
This markup should also use the add_attributes function similar to how you setup the `code__attributes`.

```
{{ add_attributes(pre__attributes) }}
```

Reply:
In addition to a `pre__attributes` we should also define a separate `pre__base_class`

Reply:
Thank you for this feedback Callin! I like how the background color only highlights the code now! So it does stand out. Before the whole component looked like "code". Now currently only the lines of "code" look like code! It was also helpful to see how attributes are defined, and then used on the component itself :).